app/lib/html_css_to_image.rb

```diff
@@ -14,13 +14,13 @@ def self.url(html:, css: nil, google_fonts: nil)
 
   def self.fetch_url(html:, css: nil, google_fonts: nil)
     cache_key = "htmlcssimage/#{html}/#{css}/#{google_fonts}"
-    cached_url = Rails.cache.read(cache_key)
+    cached_url = RedisRailsCache.read(cache_key)
```

Comment:
In order to roll this over smoothly and not deal with a bunch of misses initially, let's continue to read from Memcache BUT also write the key to Redis. After letting that bake for a few days then we can merge a second PR to begin reading from Redis and comlpetely remove our dependency on Memcache. 

What do you all think? 

@jeffin143 I want to be respectful of your time and I know issuing another PR can be time-consuming so if you would like I can do the second PR and then you can just update this one with master. Whatever you want!

Reply:
@mstruve That was really sweet of you :)

I could handle that, not an issue, So here is what I understood 
Let this line be there 
```
cached_url = Rails.cache.read(cache_key)
```
And the then add a line here [](https://github.com/thepracticaldev/dev.to/blob/d1b82dfdafe2f613513d8ef99e0d0b53d93df815/app/lib/html_css_to_image.rb#L23)
```
 RedisRailsCache.write(cache_key, image_url) unless image_url == FALLBACK_IMAGE
```

Reply:
I have raised a PR, Let me know if there are any changes required #4821